tegra/tegra.c

```diff
@@ -206,7 +206,7 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 		return -EINVAL;
 
 	if (!atomic_dec_and_test(&bo->ref))
-		return 0;
+		return atomic_read(&bo->ref);
```

Comment:
Well, I think it would be better if we'd leave that assertion-burden to the libdrm users. Applications execution abortion maybe undesirable even if something gone horribly wrong.